tidb-cloud/connect-to-tidb-cluster.md

```diff
@@ -26,7 +26,7 @@ After your TiDB cluster is created on TiDB Cloud, you can use one of the followi
 
 To connect to a Serverless Tier cluster via standard connection, perform the following steps:
 
-1. Navigate to the **Clusters** page.
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page.
```

Comment:
```suggestion
1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and choose your target project on the top of the left navigation bar.
```

When users have multiple projects, it is necessary for them to confirm or switch the project after clicking this link.